src/bridge/MatrixHandler.ts

```diff
@@ -336,6 +336,33 @@ export class MatrixHandler {
         }
     }
 
+    /**
+     * Called when a Matrix user tries to invite another user into a PM
+     * @param {Object} event : The Matrix invite event.
+     * @param {MatrixUser} inviter : The inviter (sender).
+     * @param {MatrixUser} invitee : The invitee (receiver).
+     * @return {Promise} which is resolved/rejected when the request finishes.
+     */
+    private async handleInviteToPMRoom(req: BridgeRequest, event: MatrixEventInvite,
+        inviter: MatrixUser, invitee: MatrixUser): Promise<BridgeRequestErr|null> {
+        // We don't support this
+        req.log.warn(
+            `User ${inviter.getId()} tried to invite ${invitee.getId()} to a PM room. Disconnecting from room`
+        );
+        const store = this.ircBridge.getStore();
+        const [room] = await store.getIrcChannelsForRoomId(event.room_id);
+        await store.removePmRoom(event.room_id);
+        const userId = room.server.getUserIdFromNick(room.channel);
```

Comment:
How can you infer the name of a user id from a room channel?
Is that because a PM channel in IRC has the name of the user?

Reply:
Yep. The nickname of the IRC user is the IRC channel name for PMs.

Reply:
Can they include any weird characters we may need to transform or escape?

Reply:
getUserIdFromNick should take care of that.